projects/ngx-oxford/src/lib/oxford.pipe.spec.ts

```diff
@@ -50,4 +50,19 @@ describe('OxfordPipe', () => {
     expect(() => pipe.transform(['foo'], '')).toThrowError(/is not a word/);
     expect(() => pipe.transform(['foo'], null)).toThrowError(/is not a word/);
   });
+
+  it('should truncate a list with specific conjunction', () => {
+    expect(pipe.transform(['orange', 'apple', 'mango', 'banana', 'pineapple'], 'and', {trail: 'etc', maxWords: 3}))
+          .toBe('orange, apple, mango, etc');
+  });
+
+  it('should proceed as usual if truncate word count is larger that the array length', () => {
+    expect(
+      pipe.transform(
+        ['orange', 'apple', 'mango', 'banana', 'pineapple'],
+        'and',
+        {trail: 'etc', maxWords: 12}
+      )
+    ).toBe('orange, apple, mango, banana, pineapple, etc');
```

Comment:
Sorry if I wasn't clear on this: since this doesn't go over the maxWords, it's displaying the whole list and isn't truncated. Thus it should use the conjunction, not the truncation trail: `'orange, apple, mango, banana, and pineapple'`

Reply:
Ah, sorry, it was a misunderstanding your request. I have fixed that now